.github/workflows/r-ci.yml

```diff
@@ -37,11 +37,13 @@ jobs:
       - name: Bootstrap
         run: cd apis/r && tools/r-ci.sh bootstrap
 
-      - name: Dependencies
-        run: cd apis/r && tools/r-ci.sh install_all
-
       - name: Temporarily install tiledb-r 0.20.2 from testing
         run: sudo Rscript -e 'install.packages("tiledb", repos = c("https://cloud.r-project.org", "https://tiledb-inc.r-universe.dev"))'
+        env:
+          USE_BPSM: FALSE
```

Comment:
That will let you do it in one step but will bring all dependencies of tiledb in 'raw' and not binaries so it will be slower.

Reply:
Successful on [macOS](https://github.com/single-cell-data/TileDB-SOMA/actions/runs/5743324145/job/15567348850?pr=1570#step:4:56) but not [ubuntu](https://github.com/single-cell-data/TileDB-SOMA/actions/runs/5743324145/job/15567348680?pr=1570#step:4:118).

Reply:
I may have something better.  Give it a few more minutes to finish in the other repo.

Reply:
See your slack DMs.